src/modules/likeDataApi.js

```diff
@@ -0,0 +1,24 @@
+// link to likes and message store api
+const projectID = 'cq8n4ydnTGepOHztUKmY';
+const likesLink = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${projectID}/likes/`;
+
+export const addLikeToItem = async (itemId) => {
+  const request = new Request(likesLink);
+  const response = await fetch(request, {
+    method: 'POST', // *GET, POST, PUT, DELETE, etc.
+    mode: 'cors', // no-cors, *cors, same-origin
+    body: JSON.stringify({ item_id: `${itemId}` }),
+    cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: 'same-origin', // include, *same-origin, omit
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  return response;
+};
+
+export const getLikesToItems = async () => {
+  const response = await fetch(likesLink);
+  const data = await response.json();
+  return data;
+};
```

Comment:
- [ ] the is looking as expected but this is a small issue in the like feature which is when I click on like the first time it doesn't respond until I click the second click.
this is a bug that may confuse the user so please make sure to fix it